person.rb

```diff
@@ -10,39 +10,37 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
 
   def correct_name
     @name
   end
 
   def can_use_services?
-    of_age
+    of_age? || @parent_permission
   end
 
   def parent_permission?
     @parent_permission
   end
 
+  # def rental
+  #   if @rental.empty?
+  #     "This person doesn't have any books."
+  #   else
+  #     @rental.map { |rental| rental.book.title }.join(', ')
+  #   end
+  # end
```

Comment:
- [optional] Kindly remove this commented lines of code to make your codebase clean. 👍

Reply:
@OyePriscilla  actually I noticed that but I have some plans regarding this and don't want to commit to drilling 
so I left it as it is thanks for the review